public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
+  var div = document.createElement("div");
```

Comment:
That's a fairly good point. Maybe I just picked bad names. I'll think about it again.